docs/source/application_development/http_character_control.rst

```diff
@@ -167,6 +167,6 @@ This endpoint controls the ``Enrico.encrypt_message`` method.
 - HTTP Method: ``POST``
 - Required arguments:
     - ``message`` -- encoded as base64
-- Returns: ``message_kit`` and ``signature`` encoded as base64
+- Returns: ``message_kit`` encoded as base64
```

Comment:
Required arguments for `/retrieve` (L149)  need to remove `label` and include `encrypted_treasure_map`